README.md

```diff
@@ -20,13 +20,30 @@ Project DeepSpeech is an open source Speech-To-Text engine that uses a model tra
 * [TensorFlow 1.0 or above](https://www.tensorflow.org/install/)
 * [SciPy](https://scipy.org/install.html)
 * [PyXDG](https://pypi.python.org/pypi/pyxdg)
-* [python_speech_features](https://pypi.python.org/pypi/python_speech_features)
 * [python sox](https://pypi.python.org/pypi/sox)
 * [pandas](https://pypi.python.org/pypi/pandas)
+* [DeepSpeech native client libraries](https://tools.taskcluster.net/index/artifacts/#project.deepspeech.deepspeech.native_client.master/)
 
 
 ## Install
 
+### Installing pre-built DeepSpeech Python bindings
+
+Pre-built binaries can be found on [TaskCluster](https://tools.taskcluster.net/index/artifacts/#project.deepspeech.deepspeech.native_client.master/project.deepspeech.deepspeech.native_client.master.cpu). You'll need to download `native_client.xz` and the appropriate Python wheel package.
```

Comment:
I think offering both is the way to go: "Linux 64 bits / python 2.7" and "Other targets". 
Some other question: If we anyhow go for "Linux 64 bits / python 2.7" as some kind of standard - how about an additional Debian package? This would at least lower a bit the `LD_LIBRARY_PATH` hurdle.

Another thing: All MacOS users have to go through the "build from source" procedure now, right?